windows_service/tests/test_windows_service.py

```diff
@@ -0,0 +1,74 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.win.wmi.winwmi_check import WinWMICheck
```

Comment:
Shouldn't we be using `WindowsService` and not `WinWMICheck`?

Reply:
🤦‍♂️ what was I doing...